api/src/paths/project/{projectId}/attachments/upload.ts

```diff
@@ -134,7 +134,19 @@ export function uploadAttachment(): RequestHandler {
 
       const attachmentService = new AttachmentService(connection);
 
-      const uploadResponse = await attachmentService.uploadMedia(projectId, rawMediaFile, metadata);
+      const s3Key = generateS3FileKey({
+        projectId: projectId,
+        fileName: rawMediaFile.originalname,
+        folder: S3Folder.ATTACHMENTS
+      });
+
+      const uploadResponse = await attachmentService.uploadMedia(
+        projectId,
+        rawMediaFile,
+        metadata,
+        s3Key,
+        S3Folder.ATTACHMENTS
```

Comment:
This is no longer needed right? As its being used in the generate call above already